lib/Conversion/TritonGPUToLLVM/ReduceOpToLLVM.cpp

```diff
@@ -170,37 +179,47 @@ struct ReduceOpConversion
     ReduceOpHelper helper(op);
     Location loc = op->getLoc();
     unsigned axis = op.getAxis();
+    // Specifies whether the reduce operation returns an index
+    // rather than a value, e.g. argmax, argmin, .. etc
     bool withIndex = triton::ReduceOp::withIndex(op.getRedOp());
 
     auto srcTy = op.getOperand().getType().cast<RankedTensorType>();
     auto srcLayout = srcTy.getEncoding();
     if (!helper.isSupportedLayout()) {
       assert(false && "Unexpected srcLayout in ReduceOpConversion");
     }
+    // The order of the axes for the the threads within the warp
     auto srcOrd = triton::gpu::getOrder(srcLayout);
+    // The elements owned by each thread, e.g. [2, 2] means that each thread
```

Comment:
This is not true. `getSizePerThread` is just the contiguous size. Also this comment isn't very useful as it almost just repeats the function call name. Maybe `getSizePerThread` should be renamed `getContiguousSizePerThread` and comments be there?

Reply:
I think it's clear, it wasn't confusing to me.